dist/index.js

```diff
@@ -4836,6 +4872,21 @@ async function getElixirVersion(exSpec0, otpVersion) {
     : elixirVersionWithOTP
 }
 
+async function getGleamVersion(gleamSpec0) {
+  const gleamSpec = gleamSpec0.match(/^v?(.+)/)
+  const gleamVersions = await getGleamVersions()
+  const gleamVersion = getVersionFromSpec(gleamSpec[1], gleamVersions)
+  if (gleamVersion === null) {
+    throw new Error(
+      `Requested Gleam version (${gleamSpec0}) not found in version list ` +
+        "(should you be using option 'version-type': 'strict'?)",
+    )
+  }
+
+  const digitStart = new RegExp('^\\d+')
```

Comment:
I wonder if we should make this a global const?

Reply:
Or a function, at least (?) - I'm assuming you're worried about maintainability, not performance, here.

Reply:
Yeah, definitely not performance, `const` just screams global to me, but a function is just fine.

Reply:
> `const` just screams global to me

I see... some times, but surely not mostly, to me.

Reply:
cd4bb0f